src/features/metadata-instance-editor/Instance.js

```diff
@@ -4,7 +4,7 @@ import { FormattedMessage } from 'react-intl';
 import classNames from 'classnames';
 import isEqual from 'lodash/isEqual';
 import cloneDeep from 'lodash/cloneDeep';
-import scrollIntoViewIfNeeded from 'scroll-into-view-if-needed';
+import { scrollIntoView } from '../../utils/dom';
```

Comment:
Group the import with the relative ones below, since the top section is for npm imports